conn.go

```diff
@@ -749,9 +752,10 @@ func (c *Conn) WriteMessage(messageType int, data []byte) error {
 	if err != nil {
 		return err
 	}
-	if _, err = w.Write(data); err != nil {
+	if _, err := w.Write(data); err != nil {
```

Comment:
Why this change ?

Reply:
Three months ago so I do not remember why I made this change.
There is no need to reassign here. I reverted to the original code.

Thanks.

Reply:
Also, I returned those which were unnecessary changes such as line feed.

Reply:
Thanks good job. I am really interested  in the PR, I may try it on part of our production system next week or the week after.

Reply:
I would like you to point out any improvement points you tried as a result.
I hope that good results will come!

Reply:
TBH I didn't test it yet. It's still in my todo-list but I have a lot to do for the moment, I am sorry.